soh/soh/resource/importer/SkeletonFactory.cpp

```diff
@@ -103,37 +103,41 @@ void SkeletonFactoryV0::ParseFileXML(tinyxml2::XMLElement* reader, std::shared_p
 {
     std::shared_ptr<Skeleton> skel = std::static_pointer_cast<Skeleton>(resource);
 
-    std::string skeletonType = reader->Attribute("Type");
-    // std::string skeletonLimbType = reader->Attribute("LimbType");
-    int numLimbs = reader->IntAttribute("LimbCount");
-    int numDLs = reader->IntAttribute("DisplayListCount");
-
-    if (skeletonType == "Flex") {
-        skel->type = SkeletonType::Flex;
-    } else if (skeletonType == "Curve") {
-        skel->type = SkeletonType::Curve;
-    } else if (skeletonType == "Normal") {
-        skel->type = SkeletonType::Normal;
+    skel->type = SkeletonType::Flex; // Default to Flex for legacy reasons
+    if (reader->FindAttribute("Type")) {
+        std::string skeletonType = reader->Attribute("Type");
+
+        if (skeletonType == "Flex") {
+            skel->type = SkeletonType::Flex;
+        } else if (skeletonType == "Curve") {
+            skel->type = SkeletonType::Curve;
+        } else if (skeletonType == "Normal") {
+            skel->type = SkeletonType::Normal;
+        }
     }
 
-    skel->type = SkeletonType::Flex;
```

Comment:
The diff might make it hard to see, but this was the issue, we were parsing the SkeletonType, then just re-setting it afterwards
<img width="421" alt="Screenshot 2023-11-14 at 9 46 05 AM" src="https://github.com/HarbourMasters/Shipwright/assets/7316699/24e72c1a-c8c9-4ce6-901f-3acb6f23e354">